jerry-core/ecma/operations/ecma-array-object.c

```diff
@@ -165,6 +166,8 @@ ecma_op_create_array_object_by_constructor (const ecma_value_t *arguments_list_p
   }
 
   ecma_object_t *constructor_object_p = ecma_get_object_from_value (constructor_value);
+
+  /* FIXME: Use @@species after Symbol has been implemented */
```

Comment:
Please  write `TODO` instead of `FIXME`. We are using only `TODO` comments since #990